lib/DDG/Spice/Hackage.pm

```diff
@@ -0,0 +1,16 @@
+package DDG::Spice::Hackage;
+# ABSTRACT: Search for information about Hackage packages
+
+use DDG::Spice;
+
+triggers startend => 'haskell', 'hoogle', 'hackage';
```

Comment:
Wondering if we should add "haskell packge"? I'm not sure if users are likely to have the word "package" in their query or not.

Alternatively we could strip it out of the remainder if think it's going to be there.